### DISCUSSION TITLE
Connect to Remote machine and remote machine screen too light

### DISCUSSION THREAD

**Discussion by akong77** [rustdesk/rustdesk#9076] [General]
Hello,
I upgrade from 1.2.3-2 to 1.2.7 both local and remote machine.
But when I connect to remote machine.The remote machine screen too light.
How to fix it?

**Comment by 21pages** [comment]
> The remote machine screen too light

Could you make a screenshot and try different codecs?

![20240815162417](https://github.com/user-attachments/assets/d5814a22-814d-4ed3-ad29-6558371c4ec7)

**Reply by akong77** [reply]
![圖片](https://github.com/user-attachments/assets/8b9e89de-36f9-4337-bf55-1524566ec8c8)
The font not clean.I have test connect to 1.2.3-2 is normal.

**Reply by 21pages** [reply]
Is it caused by scale original/scale adaptive?

![image](https://github.com/user-attachments/assets/bf274a76-494d-451d-bd52-41646613aa3d)

**Reply by 21pages** [reply]
Could you try 1.2.7->1.2.3-2 and 1.2.3-2->1.2.7? What are the local and remote resolution?

**Reply by akong77** [reply]
The same.Use 1920X1080

**Comment by akong77** [comment]
I use Scale adaptive.

**Reply by 21pages** [reply]
1.2.7->1.2.3-2 and 1.2.3-2->1.2.7, which one is normal?

**Reply by akong77** [reply]
1.2.7->1.2.3-2 is normal.

**Reply by akong77** [reply]
1.2.7->1.2.7 font not clean.
I have not test 1.2.3-2 -> 1.2.7

**Reply by 21pages** [reply]
Could you try different codecs with 1.2.7->1.2.7?

**Reply by akong77** [reply]
I try it.

**Reply by akong77** [reply]
VP9 is clean.

**Reply by akong77** [reply]
H264 still not clean.

**Reply by 21pages** [reply]
1.2.7->1.2.3-2, is H264 clean? What's the gpu of the controlled side?

**Reply by akong77** [reply]
Controled side (1.2.7) is Intel UHD 630.

**Reply by akong77** [reply]
> 1.2.7->1.2.3-2, is H264 clean

It's show auto(AV1),No H264

**Reply by 21pages** [reply]
You can try "Good image quality" for H264, or increate bitrate manually.

**Reply by akong77** [reply]
1.2.7->1.2.7 Control side use VP9 clean
1.2.7->1.2.3-2 Control side use AUTO(VP1) clean

**Reply by akong77** [reply]
I has increase bitrate to 100% and more.But I can't see any change.
